constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,78 @@
+mod MemoryAir
+
+### Helper functions/evaluators ###################################################################
+
+# Enforces that columns must be zero.
+ev equal_zero(&main: [4]):
+    enf v = 0 for v in &main
+
+# Enforces that values in the columns in the current row must be equal to the values in the next 
+# row.
+ev equal_seq_lines(&main: [4]):
```

Comment:
The syntax doesn't look correct here. I think we need named columns here and we shouldn't use `&`?

nit: I'm not sure I understand what the name is supposed to mean?

Reply:
I was trying to come up with some name that was supposed to express that two consecutive lines should be equal, so I came up with "equal sequential lines", which was shortened to `equal_seq_lines`. I'm happy to replace that name, but I haven't been able to come up with something better yet. Maybe `equal_cons_lines`?